alacritty/src/display/mod.rs

```diff
@@ -597,8 +516,20 @@ impl Display {
             (Vec::new(), Vec::new())
         };
 
+        // We use vsync everywhere except wayland.
+        if !is_wayland {
+            if let Err(err) =
+                surface.set_swap_interval(&context, SwapInterval::Wait(NonZeroU32::new(1).unwrap()))
+            {
+                // This is not really an error we should broadcast to the user.
```

Comment:
This comment is a bit strange, stating we shouldn't do something without any explanation.

And I'm also questioning whether the statement is even true. If vsync could not be set, it seems like something very relevant to broadcast? I'd assume this could happen when vsync was disabled in the driver? That's definitely something the user might care about, since we don't operate correctly without vsync.

Reply:
I'll keep warn for now, since the errors about us crashing due to missing vsync is close to `None`.